packages/components/src/components/post-tabs/post-tabs.tsx

```diff
@@ -0,0 +1,161 @@
+      });
+
+      // move each post-tab-header element to the "tabs" slot if it's not already there
+      if (tab.getAttribute('slot') !== 'tabs') {
+        tab.setAttribute('slot', 'tabs');
+      }
+    });
+  }
+
+  async componentDidLoad() {
+    const initiallyActivePanel = this.activePanel || this.tabs.item(0).panel;
+    await this.show(initiallyActivePanel);
+
+    this.isLoaded = true;
+  }
+
+  /**
+   * Shows the panel with the given name and selects its associated tab.
+   * Any other panel that was previously shown becomes hidden and its associated tab is unselected.
+   */
+  @Method()
+  async show(panelName: string) {
+    // do nothing if the tab is already active
+    if (panelName === this.activeTab?.panel) {
+      return;
+    }
+
+    const previousTab = this.activeTab;
+    const newTab : HTMLPostTabHeaderElement = this.host.querySelector(`post-tab-header[panel=${panelName}]`);
+    this.activateTab(newTab);
+
+    // if a panel is currently being displayed, remove it from the view and complete the associated animation
+    if (this.showing) {
+      this.showing.effect['target'].remove();
+      this.showing.finish();
+    }
+
+    // hide the currently visible panel only if no other animation is running
+    if (previousTab && !this.showing && !this.hiding) {
+      this.hidePanel(previousTab.panel);
+    }
+
+    // wait for any hiding animation to complete before showing the selected tab
+    if (this.hiding) {
+      await this.hiding.finished;
+    }
+
+    this.showSelectedPanel();
+
+    // wait for any display animation to complete for the returned promise to fully resolve
+    if (this.showing) {
+      await this.showing.finished;
+    }
+
+    this.tabChange.emit(this.activeTab.panel);
+  }
+
+  private activateTab(tab: HTMLPostTabHeaderElement) {
+    if (this.activeTab) {
+      const tabTitle = this.activeTab.shadowRoot.querySelector('.tab-title');
+      tabTitle.setAttribute('aria-selected', 'false');
+      tabTitle.classList.remove('active');
+    }
+
+    const tabTitle = tab.shadowRoot.querySelector('.tab-title');
+    tabTitle.setAttribute('aria-selected', 'true');
+    tabTitle.classList.add('active');
+
+    this.activeTab = tab;
+  }
+
+  private hidePanel(panelName: HTMLPostTabPanelElement['name']) {
+    const previousPanel = this.panelsByName.get(panelName);
+
+    this.hiding = fadeOut(previousPanel);
+    this.hiding.onfinish = () => {
+      previousPanel.remove();
+      this.hiding = null;
+    };
+  }
+
+  private showSelectedPanel() {
+    const panel = this.panelsByName.get(this.activeTab.panel);
+    this.host.appendChild(panel);
+
+    // prevent the initially selected panel from fading in
+    if (!this.isLoaded) return;
+
+    this.showing = fadeIn(panel);
+    this.showing.onfinish = () => {
+      this.showing = null;
+    };
+  }
+
+  render() {
+    return (
+      <Host data-version={version}>
+        <div class="tabs-wrapper">
+          <ul class="tabs nav" role="tablist">
```

Comment:
The semantics don't work with voice over. Voice over announces "[tab text], tab, 1 of 1". It does not recognise the other tabs as belonging to the same group. Might be due to the fact that the web component is the child of the ul and not the li. Not sure how to handle this though.

Reply:
I cannot reproduce this issue, did you use NVDA?

Reply:
The semantic works fine with NVDA, I created the ticket #1869 to find a solution with VoiceOver.